gym_cap/envs/agent.py

```diff
@@ -86,7 +90,30 @@ def move(self, action, env, team_home):
                 env[self.x, self.y] = TEAM2_UAV if self.air else TEAM2_UGV        
         else:
             print("error: wrong action selected")
+    
+    def update_memory(self):
+
+        """
+        saves/updates individual map of an agent
 
+        """
+        
+        obs = self.get_obs()
```

Comment:
Use the following:
```py
get_obs(self, com_ground, com_air, env, distance, freq, *args)
```

All parameters will have default values except env.
So, the code should be simply ```get_obs(env=env)```

We need 'env,' so add env parameter for update_memory method.

Reply:
added 'env' parameter to update_memory